measurement_files/geography_based.py

```diff
@@ -48,6 +52,66 @@ def geo_ctr(dataloader, args):
 
     pickle.dump(counts, open("results/{}/geo_ctr.pkl".format(args.folder), "wb"))
 
+def custom_geo_ctr(dataloader, args):
+    # TODO: remove process_cutoff once finalized
+    process_cutoff = 100
+
+    # import custom political boundaries, here I am using Germany's
+    # administration region political boundaries
+    with open("/Users/home/Downloads/stanford-nh891yz3147-geojson.json") as f:
```

Comment:
shouldn't this be taken in as input through the datasets.py class? rather than a hard-coded filepath

Reply:
changed